Jenkinsfile

```diff
@@ -229,28 +225,126 @@ pipeline{
                                 sh "bash -i -c './tests/testAutopas/runTests'"
                             }
                         }
-                    },
-                    "checkExamples": {
+                    }
+                )
+            }
+            post{
+                success{
+                    githubNotify context: 'test', description: currentBuild.durationString,  status: 'SUCCESS', targetUrl: currentBuild.absoluteUrl
+                }
+                failure{
+                    githubNotify context: 'test', description: currentBuild.description,  status: 'FAILURE', targetUrl: currentBuild.absoluteUrl
+                }
+                unstable{
+                    githubNotify context: 'test', description: currentBuild.description,  status: 'FAILURE', targetUrl: currentBuild.absoluteUrl
+                }
+                aborted{
+                    githubNotify context: 'test', description: 'build aborted',  status: 'ERROR', targetUrl: currentBuild.absoluteUrl
+                }
+            }
+        }
+        stage("checkExamples") {
+            steps{
+                parallel(
+                    "default": {
+                        githubNotify context: 'checkExamples', description: 'checking examples in progress...',  status: 'PENDING', targetUrl: currentBuild.absoluteUrl
                         container('autopas-gcc7-cmake-make') {
                             dir("build/examples") {
                                 sh 'ctest -C checkExamples -j8'
                             }
                         }
+                    },
+                    "gcc openmp": {
+                        container('autopas-gcc7-cmake-make') {
+                            dir("build-openmp/examples") {
+                                sh 'ctest -C checkExamples -j8'
```

Comment:
why do you use -j 4 for building but -j 8 for testing?

Reply:
* for building: all builds are done on the same machine, thus -j4 is the maximum one should do.
* for testing: here the -j doesn't have much influence, it just says how many tests can be run in parallel. We could probably reduce this number, to reduce the number of parallel things a bit, but it shouldn't change much.